extras/parse_travis_log.py

```diff
@@ -886,11 +886,12 @@ def build_return_code(status_vera_init,
     #       It may cause false positives. Even though cppcheck is executed
     #       and all its messages can be found in the log, cppcheck messages
     #       will not cause the build to fail.
-    #       To activate cppcheck, add following line to the if statement below.
-    #       Remove also the two strings '(DEACTIVATED)' behind cppcheck in
-    #       the summary_table above.
+    #       To activate cppcheck, add following lines to the if statement
+    #       below. Remove also the two strings '(DEACTIVATED)' behind
+    #       cppcheck in the summary_table above.
     #
     #       (status_cppcheck_init is None or status_cppcheck_init) and \
+    #       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
I actually think that there is a benefit in running checks and having their output in the log  but ignoring it for determining the build status.

However, I agree with @heplesser that hiding this in a comment might not be the best solution. Another approach would be to explicitly have `skipping_<checker>` flags for all style checkers and use those in l.902ff. The `(deactivated)` printout could then also be be added automatically (and called `ignored`).